nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -46,6 +46,7 @@ let
         binary-caches = ${toString cfg.binaryCaches}
         trusted-binary-caches = ${toString cfg.trustedBinaryCaches}
         binary-cache-public-keys = ${toString cfg.binaryCachePublicKeys}
+        auto-optimise-store = ${toString cfg.autoOptimiseStore}
```

Comment:
This causes nixops deploy to fail:

```
gitlab............> error: configuration option ‘auto-optimise-store’ should be either ‘true’ or ‘false’, not ‘’
gitlab............> error: configuration option ‘auto-optimise-store’ should be either ‘true’ or ‘false’, not ‘’
gitlab............> unable to connect to ‘root@git.mayflower.de’; falling back to old closure copying method
gitlab............> error: configuration option ‘auto-optimise-store’ should be either ‘true’ or ‘false’, not ‘’
gitlab............> Died at /nix/store/pjk50qzcs5n3x43q8hb4556g4xm22rr3-nix-1.11.7/lib/perl5/site_perl/5.22.2/x86_64-linux-thread-multi/Nix/CopyClosure.pm line 100.
```

I have a fix that I will push in a few minutes.